mypy/main.py

```diff
@@ -114,13 +114,16 @@ def main(script_path: Optional[str],
         code = 2 if blockers else 1
     if options.error_summary:
         if messages:
-            n_errors, n_files = util.count_stats(messages)
+            n_errors, n_notes, n_files = util.count_stats(messages)
```

Comment:
I don't think the outer `if` remains necessary. If `messages` is empty, then this returns `(0, 0, 0)`. So we can just always count the stats and then condition on them.

Also, is `n_notes == len(messages)` actually different from `not n_errors`? Is there a third type of message that is neither an error nor a note?

Reply:
There is a test case [testErrorSummaryOnBadUsage], which generates 'mypy: cannot read file 'missing.py': No such file or directory'

Reply:
Okay, I still think the comment about not needing the outer `if` applies.